spec/controllers/generic_files_controller_spec.rb

```diff
@@ -15,6 +15,13 @@
       let(:mock) { GenericFile.new(id: 'test123') }
       let(:batch) { Batch.create }
       let(:batch_id) { batch.id }
+      let(:collection) {
+        Collection.new(title: "test collection").tap do |c|
```

Comment:
You don't need the `#tap` here because `#new` takes a block. You can also use `#create` here for fewer LOC:

```ruby
Collection.create(title: 'test collection') do |c|
  c.apply_depositor_metadata(user.user_key)
end
```